ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -148,13 +148,34 @@ Item {
             height: root.showAllAccounts ? implicitHeight : 61
             walletStore: RootStore
             networkConnectionStore: root.networkConnectionStore
+            isCommunityOwnershipTransfer: (!!walletStore.currentViewedCollectible && walletStore.currentViewedHoldingID !== "") ?
+                                              (walletStore.currentViewedCollectible.communityPrivilegesLevel === Constants.TokenPrivilegesLevel.Owner) : false
```

Comment:
I'd recommend also checking `walletStore.currentViewedCollectible.communityId !== ""`, which is how we determine if it's a Community-minted collectible or just some normal one.